cmd/clusterctl/hack/create-local-repository.py

```diff
@@ -209,6 +209,8 @@ def splitNameAndType(provider):
         return provider[len('ipam-'):], 'IPAMProvider'
     if provider.startswith('runtime-extension-'):
         return provider[len('runtime-extension-'):], 'RuntimeExtensionProvider'
+    if provider.startswith('add-on-'):
```

Comment:
```suggestion
    if provider.startswith('addon-'):
```
same in the line below